.github/workflows/pushing_container_image.yml

```diff
@@ -2,49 +2,32 @@ name: Publish image
 on:
   release:
     types: [published]
+
 jobs:
   push_to_registries:
     name: Push image to multiple registries
     runs-on: ubuntu-latest
     steps:
-      -
-        name: Checkout code
+      - name: Checkout code
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      -
-        name: Set up Docker Buildx
+
+      - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
         with:
           buildkitd-flags: --debug
-      -
-        name: Login to ghcr.io
-        uses: docker/login-action@v1
-        with:
-          registry: ghcr.io
-          username: ${{ github.repository_owner }}
-          password: ${{ secrets.GITHUB_TOKEN }}
-      
-      - name: lowercase Repo
-        run: |
-            echo "REPO_OWNER=${GITHUB_REPO_OWNER,,}" >>${GITHUB_ENV}
-        env:
-          GITHUB_REPO_OWNER: ${{ github.repository_owner }}
 
+      - name: Login to ghcr.io
+        run: |
+          echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
 
```

Comment:
The login change now creates this warning:
> WARNING! Your password will be stored unencrypted in /home/runner/.docker/config.json.
Configure a credential helper to remove this warning. See
https://docs.docker.com/engine/reference/commandline/login/#credentials-store

ok? Or revert to `docker/login-action@v1` ?

Reply:
Argh! Thanks for spotting. That's not good at all. I'll revert - I did have problems with it AFAIR, but we can't have that leak.